gridcapa-task-manager-app/src/main/java/com/farao_community/farao/gridcapa/task_manager/app/TaskManagerController.java

```diff
@@ -87,4 +92,20 @@ private ResponseEntity<byte[]> getZippedGroup(OffsetDateTime timestamp, String f
     public ResponseEntity<List<TaskDto>> getListTasksFromBusinessDate(@PathVariable String businessDate) {
         return ResponseEntity.ok().body(builder.getListTasksDto(LocalDate.parse(businessDate)));
     }
+
+    @GetMapping(value = "/file/{fileType}/{timestamp}", produces = "application/octet-stream")
+    public @ResponseBody Object getFile(@PathVariable String fileType, @PathVariable String timestamp) throws IOException {
```

Comment:
Return ResponseEntity<byte[]> to remain consistent with other controller methods. You can get inspired by getZippedGroup method to return HTTP response for the file, so you have proper filename set up for downloading.

Reply:
Now return a ResponseEntity<byte[]>